src/bicep-registry/scheduled-action/main.bicep

```diff
@@ -10,31 +10,40 @@ param name string
 @description('Indicates the kind of scheduled action. Default: Email.')
 @allowed([
   'Email'
-  'InsightAlert' // @subscription
+  /* @subscription
```

Comment:
but it would get uncommented when building the appropriate scope. My thinking is, no matter what, all scope specific logic should be commented for easy parsing. Or maybe I'm missing something altogether with these tags